storyblok/field-plugin
# feat(lib): add event listener for ESC key handler

## What?

Whenever a modal is opened inside a custom field plugin, it is impossible to close the modal on escape. This keyboard listener fixes that issue inside and outside the iframe by changing the isModalOpen state and notifying the container.

## Why?

JIRA: SHAPE-2151

This is important for people who use a keyboard only to navigate.

## How to test? (optional)

1. Run any template locally
2. Go to the field-plugin editor at Partner Portal (Enable local dev mode)
3. Open the modal, put the focus on something inside the iframe
4. Press ESC, see the modal closing
